src/api/app/views/webui/request/_actions_details.html.haml

```diff
@@ -10,6 +19,7 @@
         %button.btn.btn-outline-primary.btn-sm.dropdown-toggle.rounded-0{ 'aria-expanded' => 'false', 'data-bs-toggle' => 'dropdown',
                                                                           :type => 'button', 'data-boundary' => 'viewport' }
           Select Action
+          %i.fas{ class: quick_build_results_status(buildresults) }
```

Comment:
I wouldn't display it here in the dropdown header, it does not belong the the whole Request and this is only to select one action. I'd rather move it under the `Action details` part, as it belongs to that one. Something roughly like the following:
![image](https://user-images.githubusercontent.com/7080830/229122753-34982421-213e-4236-b200-c00d9b47ee9f.png)